src/main/java/seedu/address/model/appointment/Appointment.java

```diff
@@ -32,17 +32,21 @@ public class Appointment {
     private final LocalDateTime dateTime;
     private final Person patient;
 
+    private final String doctorName;
+
+
     /**
      * Constructor for Appointment instance
      * @param description Description of the appointment
      * @param dateTime Time and Date of the appointment
      */
-    public Appointment(String description, LocalDateTime dateTime, Person patient) {
+    public Appointment(String description, LocalDateTime dateTime, Person patient, String name) {
```

Comment:
Can you be more specific with the "name"? Perhaps like nameOfDoctor or something. This makes me think appointment has a name on a first glance